File: CS/Snap_API/CategoryForm.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace Snap_API {
    public partial class CategoryForm : DevExpress.XtraEditors.XtraForm {
        public CategoryForm() {
            InitializeComponent();
        }
        public string Role {
            get {
                return cbCategory.EditValue.ToString();
            }
            
        }
    }
}

File: VB/Snap_API/CategoryForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq

Namespace Snap_API
	Partial Public Class CategoryForm
		Inherits DevExpress.XtraEditors.XtraForm
		Public Sub New()
			InitializeComponent()
		End Sub
		Public ReadOnly Property Role() As String
			Get
				Return cbCategory.EditValue.ToString()
			End Get

		End Property
	End Class
End Namespace